Cython/Compiler/Scanning.py

```diff
@@ -330,6 +330,9 @@ def __init__(self, file, filename, parent_scanner=None,
         self.indentation_char = None
         self.bracket_nesting_level = 0
         self.async_enabled = 0
+        if parent_scanner and hasattr(parent_scanner, 'async_enabled') and \
+            parent_scanner.async_enabled:
```

Comment:
Is this whole `hasattr` dance really needed? Why wouldn't a normal attribute access work?
(BTW, `getattr()`with default is more efficient than `hasattr()` plus an attribute lookup.)

Reply:
Like I saw from Travis's tests not every parent scaner has attribute "async_enabled" (like I undeerstood ppoblem was in some anaconda's builds), about other - ok.

Reply:
Compilation turns the `async_enabled` attribute into an internal, compiled attribute (since it's declared as a `cdef` attribute). You can make it `cdef readonly` in `Scanning.pxd` to make it available as a read-only Python property, and then just copy its value over from the `parent_scanner`, instead of calling `enter_async()`.

Reply:
ok, thanks